### PR TITLE
Warn user when adding instance to layer that is not visible

### PR DESCRIPTION
**Pull request by AlexandreSi**

### CODE REVIEW

File: newIDE/app/src/SceneEditor/index.js [hunk 1]
```diff
@@ -1825,6 +1844,23 @@ export default class SceneEditor extends React.Component<Props, State> {
             </React.Fragment>
           )}
         </I18n>
+        {this.state.invisibleLayersOnWhichInstancesHaveJustBeenAdded && (
+          <InfoBar
+            message={
+              <Trans>
+                You just added an instance to the layers
```
Comment (ClementPasteau) [inline newIDE/app/src/SceneEditor/index.js:1851]: should we add a ⚠️  at the beginning?

File: newIDE/app/src/SceneEditor/index.js [hunk 2]
```diff
@@ -514,7 +517,14 @@ export default class SceneEditor extends React.Component<Props, State> {
   };
 
   _onInstancesAdded = (instances: Array<gdInitialInstance>) => {
+    const invisibleLayersOnWhichInstancesHaveJustBeenAdded = [];
     instances.forEach(instance => {
+      const layer = this.props.layout.getLayer(instance.getLayer());
+      if (!layer.getVisibility()) {
+        invisibleLayersOnWhichInstancesHaveJustBeenAdded.push(
```
Comment (ClementPasteau) [inline newIDE/app/src/SceneEditor/index.js:524]: is there a risk you add the same layer multiple times in this array?

File: newIDE/app/src/SceneEditor/index.js (hunk 1 above)
Comment (ClementPasteau) [reply newIDE/app/src/SceneEditor/index.js:1851]: Can we add to multiple layers at the same time? 🤔 
I'd simplify the sentence to `You just added an instance to a hidden layer (xxx). Open the layer panel to make it visible`

Comment (4ian) [inline newIDE/app/src/SceneEditor/index.js:1851]:
```suggestion
                You just added an instance to the layer(s)
```

**Review by 4ian — APPROVED** [review]

File: newIDE/app/src/SceneEditor/index.js (hunk 2 above)
Comment (4ian) [reply newIDE/app/src/SceneEditor/index.js:524]: Good question!

File: newIDE/app/src/SceneEditor/index.js (hunk 1 above)
Comment (4ian) [reply newIDE/app/src/SceneEditor/index.js:1851]: I say:
- I agree on simplifying the sentence (honestly, nobody will make the error multiple time, we warn about the last layer, it's good enough). And so probably simplify the state to contain the last layer.
  Less is more. With the plural, the sentences is getting hard to parse.
- No ⚠️. If we want to add this, it should be a prop and part of the design of the app, not an emoji.
